.github/actions/on_host_tests/action.yaml

```diff
@@ -39,6 +39,10 @@ runs:
         LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${out_dir}"
         export LD_LIBRARY_PATH
 
+        # Set shard info for test targets that doesn't use base.
+        export GTEST_TOTAL_SHARDS=${{ matrix.shard }}
+        export GTEST_SHARD_INDEX=${{ inputs.num_gtest_shards }}
+
```

Comment:
This doesn't seem to work. Something is popping off the environment variables before they reach gtest.

Logs from modified gtest:

```
$ export GTEST_SHARD_INDEX=1; export GTEST_TOTAL_SHARDS=10; out/linux-x64x11-modular_devel/nplb_loader
[nplb_loader/3371579:0722/173528.158027(UTC):INFO:audio_sink_type_dispatcher.cc(34)] Use ALSA
kTestTotalShards: -1
kTestShardIndex: -1
[==========] Running 3901 tests from 215 test suites.
[----------] Global test environment set-up.
[----------] 5 tests from SbAlignTest
[ RUN      ] SbAlignTest.AlignAsStructFieldOnStack
```